medusa/process_tv.py

```diff
@@ -334,11 +334,9 @@ def validateDir(self, path, dirName, nzbNameOriginal, failed):
         :param result: Previous results
         :return: True if dir is valid for processing, False if not
         """
-        dirName = ss(dirName)
-
-        IGNORED_FOLDERS = [u'.AppleDouble', u'.@__thumb', u'@eaDir']
+        ignored_folders = [u'.AppleDouble', u'.@__thumb', u'@eaDir']
```

Comment:
There is no such thing as a method constant, this will probably end up being a class attribute in the end.